lesson16/index.js

```diff
@@ -0,0 +1,128 @@
+// const url = "https://myjson.dit.upm.es/api/bins/hyv9";
+const url = "https://myjson.dit.upm.es/api/bins/d9vx";
+const newsBlock = document.querySelector(".news");
+const newsTabs = document.querySelector(".news_Tabs");
```

Comment:
IMO
I think it's better to use id and getElementId here.
The reason is that I think it's safer.

Reply:
[15b53e9](https://github.com/hippopo-moo/js-handson-work/pull/18/commits/15b53e939355b275395c172b9aff6120fa724a25)

It's true that getElementById is better for retrieving elements that only have one element, so I fixed it.